src/compiler/types.ts

```diff
@@ -3389,6 +3395,8 @@
     /* @internal */
     export interface TsConfigOnlyOption extends CommandLineOptionBase {
         type: "object";
+        optionDeclarations?: CommandLineOption[];
+        extraKeyDiagnosticMessage?: DiagnosticMessage;

```

Comment:
Hmm.. currently we report unknown compiler option | unknown type acquisition option. I tried to keep the same behavior. Is it ok to change the behavior?